ts/packages/cli/src/commands/export.ts

```diff
@@ -156,6 +161,12 @@ export default class Export extends BaseCommand<typeof Export> {
     this.printStdout(formattedConfigs);
   }
 
+  keysMutations() {
+    return this.flags.prefix || this.flags.suffix
+      ? (key: string) => `${this.flags.prefix ?? ''}${key}${this.flags.suffix ?? ''}`
+      : undefined;
+  }
```

Comment:
I like the strategy of having a builder function for `keys` but I think we can make this syntax a bit more extensible by going for something like this:
```node.js
  keysMutations() {
    return (key: string) => {
      let mutatedKey = key;
      if (this.flags.prefix) mutatedKey = `${this.flags.prefix}${mutatedKey}`;
      if (this.flags.suffix) mutatedKey = `${mutatedKey}${this.flags.suffix}`;
      return mutatedKey;
    };
  }
```

This will allow us to add any future mutation flags easily without having to refactor this code. What do you think?

Reply:
This will cause `ExportCommand` to **always** run the keys function. I want `keysMutations` to return undefined when no mutations needed.
How about this:
```typescript
  keysMutations() {
    if (![this.flags.prefix, this.flags.suffix].some((value) => !!value)) return undefined;
    return (key: string) => {
      let mutatedKey = key;
      if (this.flags.prefix) mutatedKey = `${this.flags.prefix}${mutatedKey}`;
      if (this.flags.suffix) mutatedKey = `${mutatedKey}${this.flags.suffix}`;
      return mutatedKey;
    };
  }
```